codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -270,9 +292,31 @@ private static boolean isClosed(Http2Stream stream) {
     }
 
     /**
+     * Flow control state that does autorefill of the flow control window when the data is
+     * received.
+     */
+    private final class AutoRefillState extends DefaultState {
+        public AutoRefillState(Http2Stream stream, int initialWindowSize) {
+            super(stream, initialWindowSize);
+        }
+
+        @Override
+        public void receiveFlowControlledFrame(int dataLength) throws Http2Exception {
+            super.receiveFlowControlledFrame(dataLength);
+            super.consumeBytes(dataLength);
```

Comment:
Actually this is right ... we don't want to call `this.consumeBytes` since it does nothing.